subiquity/common/filesystem/manipulator.py

```diff
@@ -270,10 +270,10 @@ def clear(self, obj, wipe=None):
 
     def reformat(self, disk, ptable=None, wipe=None):
```

Comment:
Note that using the default value for `ptable` will still have a behavior on disks that have partitions vs on disks that have no partition.

For disks with partitions, `disk.ptable` will be reset to `None`
For disks without partition, `disk.ptable` will be left untouched.

This feels inconsistent.